ONSdigital/sdc-int-rh-ui
# Socint 266 use available fulfilment from survey

# Motivation and Context
Switches to use new /surveys endpoint from RHSvc to determine available deliveryMethods for the case(survey), and replaces use of the /fulfilments endpoint for determining appropriate packCode

# What has changed
Adds dynamic function to determine available delivery methods for a fulfilment, determined by the underlying survey data
Adds fallback if no allowableFulfilments provided
Replaces use of /fulfilments endpoint with /survey endpoint to determine 'packCode' for call to RM to request a fulfilment

# How to test?
Start at /en/request/access-code/enter-address/
Proceed through as normal.
Check 'Select delivery method' options are driven by survey
Check appropriate packCode selection is made

# Links
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-266
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-269
